packages/components/src/draggable/README.md

```diff
@@ -53,12 +53,13 @@ const MyDraggable = () => (
 				>
 				{
```

Comment:
This "{" and the content inside should be indented one more tab. Given that we are changing this file, would it be possible also to fix this?